seatunnel-e2e/seatunnel-flink-connector-v2-e2e/connector-jdbc-flink-e2e/src/test/java/org/apache/seatunnel/e2e/flink/v2/jdbc/JdbcSourceToConsoleIT.java

```diff
@@ -99,4 +101,10 @@ public void closePostgreSqlContainer() {
             psl.stop();
         }
     }
+
+    @Override
+    protected void executeExtraCommands(GenericContainer<?> container) throws IOException, InterruptedException {
```

Comment:
For subclasses, you don't need to consider if you write this command, as long as you provide http-link,
and it's may be` static List<String> ` in subclasses.

Reply:
All of the jars in plugins/*/*.jar will be added to the lib for spark/flink

Reply:
> 

In my opinion, not only will the container download additional jar packages, but it may also execute other commands, such as creating files, etc....

Reply:
@TyrantLucifer  pls review again